src/views/molecules/search-bar/search-bar.js

```diff
@@ -9,7 +9,7 @@ import './search-bar.css';
 class SearchBar extends Component {
   constructor(props){
     super(props)
-    this.debouncedOnQueryChange = debounce(this.props.onQueryChange, 50);
+    this.debouncedOnQueryChange = debounce(this.props.onQueryChange, 0);
```

Comment:
Should ensure it would not impact on the query performance in case of stress (if it related to firebase, otherwise ignore this comment)